front/src/applications/editor/tools/pointEdition/components.tsx

```diff
@@ -340,6 +341,8 @@ export const PointEditionLeftPanel: FC<{ type: EditoastType }> = <Entity extends
           </button>
         </div>
       </EditorForm>
+
+      <EntityError className="mt-1" entity={state.entity} />
```

Comment:
Do you need a isNew test like in rangeEdition/components ? You should have the same logic everywhere (either test isNew everywhere, or don't test everywhere)
```suggestion
      {!isNew && <EntityError className="mt-1" entity={entity} />}
```